packages/utils/src/index.ts

```diff
@@ -44,7 +43,7 @@ export const insertStyles = (
   }
   if (cache.inserted[serialized.name] === undefined) {
     let stylesForSSR = ''
-    let current = serialized
+    let current: SerializedStyles | undefined = serialized
```

Comment:
Is this even actually possible for `serialized` to be `undefined` here? I would've expected `if (cache.inserted[serialized.name] === undefined) {` to error from accessing `undefined.name` if so.

If that's actually possible, would it make sense to introduce optional chaining into the if statement's syntax?

Reply:
I think it is `current`-specific typing in advance because `current.next` is possibly undefined and we're checking against it as part of the `do`/`while`:

https://github.com/emotion-js/emotion/blob/3f66787ad41923515ae72d8b03d881a3b83efdd6/packages/utils/src/index.ts#L46-L58

Reply:
Ah, gotcha, that makes sense. I haven't seen a while loop like this _in a while_ :pun-dog: so it threw me. Thanks for the quick update!